app/assets/stylesheets/elements/_codemirror.scss

```
@@ -303,4 +303,27 @@
       color: $text-color-light;
     }
   }
+
+  .cm-codeActionsLightbulb {
+    position: absolute;
+    aspect-ratio: 1;
+    line-height: 1;
+    font-size: 0.75rem;
+    background-color: $bg-darker;
+    padding: 0.25rem;
+    border-radius: 2px;
+    cursor: pointer;
+    transform: translate(-100%, -50%);
```

Everywhere else we use the longer value
```suggestion
    transform: translateX(-100%) translateY(-50%);
```